_glossary/service.md

```diff
@@ -2,6 +2,6 @@
 title: Service
 type: markdown
 ---
-A unit of an application with a unique name that other services
-use to refer to the functionality being called. Service instances are
-pods/VMs/containers that implement the service.
+A delineated group of related behaviors within a **Service Mesh**. Services are named using a **Service Name**,
+and Istio policies such as load balancing and routing are applied to **Service Names**.
+A **Service** is typically materialized by one or more **Service Endpoints**.
```

Comment:
, and may consists of multiple co-existent service versions.

Reply:
Done